src/Documenter.jl

```diff
@@ -462,12 +598,22 @@ the generated html. The following entries are valied in the `versions` vector:
    The second argument can be `"v^"`, to point to the maximum version docs
    (as in e.g. `"stable" => "v^"`).
 
+**uploader** determines _which_ CI service will upload the documentation.  Defaults to Travis
+(`TRAVIS`).  Other options are `GITLAB_CI`, `CIRRUS_CI`, `DRONE`, and `APPVEYOR`.
+
+**tag** provides a specific tag, if you want one to be set.  Providing an empty string (if
+building manually) will make the documentation build as if there was no tag; but, if run on
+CI, it will read the appropriate environment variable to determine if a tag exists or not;
+if there is a provided tag, then that will take precedence over the environment variable,
+which will take precedence over the empty string.
```

Comment:
In what situation do you want to have `tag` be set when running on CI?

Reply:
In case I want to build a tag associated with a different package - say MakieGallery building documentation for Makie.  They're versioned differently.  It's the same, I imagine, for any package whose docs are hosted externally.